storage/libradb/src/lib.rs

```diff
@@ -807,6 +743,96 @@ impl LibraDB {
 }
 
 impl DbReader for LibraDB {
+    fn update_to_latest_ledger(
+        &self,
+        client_known_version: Version,
+        request_items: Vec<RequestItem>,
+    ) -> Result<(
+        Vec<ResponseItem>,
+        LedgerInfoWithSignatures,
+        ValidatorChangeProof,
+        AccumulatorConsistencyProof,
+    )> {
+        Self::update_to_latest_ledger(self, client_known_version, request_items)
+    }
```

Comment:
possible to split these newly added interfaces to a separate commit / PR? (and how are they covered by tests?)